### PR TITLE
DOC update man pages with new CivisML features and defaults

### PR DESCRIPTION
**Pull request by elsander**
This PR updates the `civis_ml` documentation with new features and defaults from CivisML 2.1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 ### Added
 
+- Update `civis_ml` documentation with CivisML 2.1 features and defaults.
```
Comment (patr1ckm) [inline CHANGELOG.md]: I think we should briefly summarize what they are so it's easy to see at a glance.

**Comment by elsander** [comment]
Sorry for the temporary assignment there @stephen-hoover-- too used to tagging you for everything!

@patr1ckm, I updated the changelog to list the changes that I specifically mention in the documentation. The changelog for the 2.1 release is long and detailed, so I don't think it makes sense to mention it all here.

**Comment by patr1ckm** [comment]
LGTM after resolving `CHANGELOG`

**Comment by elsander** [comment]
Fixed the conflict. Reapprove?

**Pull request merged**